schema/aiconfig.schema.json

```diff
@@ -174,27 +191,6 @@
         }
       ]
     },
-    "Attachment": {
```

Comment:
hmm, why the change to make this inline instead of its own definition?

Reply:
I'm not sure -- this was autogenerated schema based on the types.ts

Reply:
It looks like we may have manually updated the schema last time, and this is being updated with the `genSchema` script.

Reply:
Ah, that was definitely me. I didn't realize there's a gen script. We should update the README to reference that and the process -- just make changes to types.ts and run `yarn gen-schema`?

Reply:
I guess the gen script removes the definition/ref for some reason. Not a big deal